docs/docs/cloud/how-tos/stream_debug.md

```diff
@@ -0,0 +1,162 @@
+# How to stream debug events
+This guide covers how to stream debug events from your graph (`stream_mode="debug"`).
+
+First let's set up our client and thread:
+
+=== "Python"
+    
+    ```python
+    from langgraph_sdk import get_client
+
+    client = get_client()
+    # create thread
+    thread = await client.threads.create()
+    print(thread)
+    ```
+
+=== "Javascript"
+
+    ```js
+    import { Client } from "@langchain/langgraph-sdk";
+
+    const client = new Client({apiUrl:"whatever-your-deployment-url-is"});
```

Comment:
general note: let's make sure there is parity between python and js code, so if we're passing custom URL here, we should do so in python too

Reply:
Are these automatically linted/formatted? If not, convention is a space after a colon, so something like

```js
const client = new Client({ apiUrl: "whatever-your-deployment-url-is" });
```